core-ui/src/components/Predefined/List/ConfigMaps.list.js

```diff
@@ -14,5 +16,20 @@ export const ConfigMapsList = ({ DefaultRenderer, ...otherParams }) => {
     },
   ];
 
-  return <DefaultRenderer customColumns={customColumns} {...otherParams} />;
+  const description = (
+    <Trans i18nKey="config-maps.description">
+      <Link
+        className="fd-link"
+        url="https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-configuration-parameters/svls-02-environment-variables#define-environment-variables-in-a-config-map"
```

Comment:
In case of plain k8s resource shouldn't we link to k8s wiki instead of Kyma wiki?